### PR TITLE
Add -Region parameter to Wait-GceManagedInstanceGroup

### PR DESCRIPTION
**Pull request by quoctruong**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/604)

### CODE REVIEW

File: Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs [hunk 1]
```diff
@@ -1335,5 +1365,25 @@ protected override void ProcessRecord()
...
+            IList<ManagedInstance> instances = request.Execute().ManagedInstances;
+            while (instances != null && instances.Any(i => i.CurrentAction != "NONE") && !Stopping)
+            {
+                if (Timeout >= 0 && stopwatch.Elapsed.Seconds > Timeout)
+                {
+                    WriteWarning("Wait-GceManagedInstanceGroup timed out for " +
+                                 $"{request.Project}/{request.Region}/{request.InstanceGroupManager}");
+                    break;
+                }
+                Thread.Sleep(150);
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs]: No need to address here but should we make this configurable?

File: Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs [hunk 2]
```diff
@@ -1335,5 +1365,25 @@ protected override void ProcessRecord()
                 instances = request.Execute().ManagedInstances;
             }
         }
+
+        private void WaitRegionManagedInstance(string project, string region, string name)
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs]: Suggestion: Create a common function for this and the above that takes a ClientServiceRequest and a string message to print while waiting.  The bulk of the logic is shared.

File: Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs (hunk 1 above)
Comment (quoctruong) [reply Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs]: I think we can either add another parameter for that or set a config for the whole module. Nobody has asked for this before so I'm not sure how useful it is though.

**Comment by quoctruong** [comment]
@iantalarico PTAL.

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**